subset/security/tlsfaux/absolute_filepath.py

```diff
@@ -0,0 +1,5 @@
+import os
```

Comment:
What's this file for?  Where is it called/used?

Reply:
this file is used in generate_certs.py and server.py > its to make python use the tlsfaux directory where *.py files exist